src/Symfony/Component/Ldap/Adapter/AbstractConnection.php

```diff
@@ -24,22 +24,51 @@
     public function __construct(array $config = array())
     {
         $resolver = new OptionsResolver();
+
+        $this->configureOptions($resolver);
+
+        $this->config = $resolver->resolve($config);
+    }
+
+    /**
+     * Configures the adapter's options.
+     *
+     * @param OptionsResolver $resolver An OptionsResolver instance
+     */
+    protected function configureOptions(OptionsResolver $resolver)
+    {
         $resolver->setDefaults(array(
-            'host' => null,
+            'host' => 'localhost',
             'port' => 389,
             'version' => 3,
-            'useSsl' => false,
-            'useStartTls' => false,
-            'optReferrals' => false,
+            'connectionString' => null,
```

Comment:
True. I'll change this for optReferrals too, which is in the ExtLdap-specific code.